core/contracts/oracle/implementation/Governor.sol

```diff
@@ -94,6 +96,10 @@ contract Governor is MultiRole, Testable {
         // Initialize the transaction array.
         for (uint256 i = 0; i < transactions.length; i++) {
             require(transactions[i].to != address(0), "The to address cannot be 0x0");
+            // If the transaction has any data with it the recipient must be a contract, not an EOA.
+            if (transactions[i].data.length > 1) {
```

Comment:
Shouldn't this be `>0`?

Reply:
We represent no data like this: `web3.utils.hexToBytes("0x0")`. This has a length of 1, not zero. It's a bit nasty. I can add a comment

Reply:
Is there a way to pass in a bytes array of length 0?

Reply:
Ye, that's a good point. I changed this to use `web3.utils.hexToBytes("0x")` which is a bytes array of length 0. I have updated the contracts accordingly to check for any length `>0` for a tx with data. is cleaner now, thanks for the suggestion 👍.